__tests__/app.test.js

```diff
@@ -231,3 +231,71 @@ describe("POST /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("PATCH /api/articles/:article_id", () => {
+  test("200; Increment votes by 1 and responds with updated article", () => {
```

Comment:
Try to keep the test description a bit more vague. If you just say "increments votes" this is fine. The test description could lead someone to believe that you can only increment by 1! 😄